packages/smart-contracts/scripts-create2/verify-one.ts

```diff
@@ -27,6 +27,8 @@ export async function VerifyCreate2FromList(hre: HardhatRuntimeEnvironmentExtend
         switch (contract) {
           case 'EthereumProxy':
           case 'EthereumFeeProxy':
+          case 'ETHConversionProxy':
```

Comment:
Not related to your PR, but your changes made me think about it: modules are called `deploy-one.ts` and `verify-one.ts` but they contain methods to deploy and verify all contracts (not just one). I guess we should rename them to `deploy.ts` and `verify.ts`.

Reply:
in scripts or scripts-create-2 or both ?

Reply:
Yes, for `scripts-create2` 🙂